code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -576,6 +576,8 @@ var/list/robot_verbs_default = list(
 /mob/living/silicon/robot/restrained()
 	return 0
 
+/mob/living/silicon/robot/CanSuccumb()
+	return low_power_mode

```

Comment:
I think this also needs to return `InCritical()`

Reply:
If(low_power_mode || InCritical())
     return TRUE

Reply:
Can borgs even go into critical? And if so is it like normal critical? I assumed they just die outright, and allowing Cyborgs to succumb in critical seems unnecessary, unless I'm wrong about this.

Reply:
Doesn't seem like it, though if borgs can't go into critical it would be better if instead of defining an CanSuccumb proc, make the borgs InCritical proc return low power mode

Reply:
Other than having their battery removed, I don't think there's a point where cyborgs are permanently, completely, helpless without outside intervention. I pretty sure that no matter how much damage they take, Cyborgs can still move up until the point they die. At this point, the succumb verb is pretty unnecessary since they either die from outside sources or can scoot away somewhere for repairs. Giving them the ability to succumb while heavily damaged seems pointless. Unless I'm wrong about this, I see no reason to make that change.

The point of this PR was to give cyborg's that have had their battery removed by antag, but not outright killed, a way to escape being stuck in permanent unconsciousness, that isn't just ghosting which makes them unable to be recovered.

Reply:
I just meant rename `CanSuccumb()` to `InCritical()` and handle borg low power mode as crit for the case of succumbing because that checks for crit